e2e/fixtures/constants.ts

```diff
@@ -0,0 +1,462 @@
a=i>1?n[i-1]:o,s=i>2?n[2]:o;for(a=e.length>3&&"function"==typeof a?(i--,a):o,s&&bi(n[0],n[1],s)&&(a=i<3?o:a,i=1),t=_e(t);++r<i;){var c=n[r];c&&e(t,c,r,a)}return t}))}function Mo(e,t){return function(n,r){if(null==n)return n;if(!Ka(n))return e(n,r);for(var o=n.length,i=t?o:-1,a=_e(n);(t?i--:++i<o)&&!1!==r(a[i],i,a););return n}}function jo(e){return function(t,n,r){for(var o=-1,i=_e(t),a=r(t),s=a.length;s--;){var c=a[e?s:++o];if(!1===n(i[c],c,i))break}return t}}function zo(e){return function(t){var n=cn(t=ws(t))?gn(t):o,r=n?n[0]:t.charAt(0),i=n?Eo(n,1).join(""):t.slice(1);return r[e]()+i}}function Uo(e){return function(t){return zt(Qs(qs(t).replace(et,"")),e,"")}}function Lo(e){return function(){var t=arguments;switch(t.length){case 0:return new e;case 1:return new e(t[0]);case 2:return new e(t[0],t[1]);case 3:return new e(t[0],t[1],t[2]);case 4:return new e(t[0],t[1],t[2],t[3]);case 5:return new e(t[0],t[1],t[2],t[3],t[4]);case 6:return new e(t[0],t[1],t[2],t[3],t[4],t[5]);case 7:return new e(t[0],t[1],t[2],t[3],t[4],t[5],t[6])}var n=$n(e.prototype),r=e.apply(n,t);return ts(r)?r:n}}function Do(e){return function(t,n,r){var i=_e(t);if(!Ka(t)){var a=li(n,3);t=Ps(t),n=function(e){return a(i[e],e,i)}}var s=e(t,n,r);return s>-1?i[a?t[s]:s]:o}}function Fo(e){return oi((function(t){var n=t.length,r=n,a=qn.prototype.thru;for(e&&t.reverse();r--;){var s=t[r];if("function"!=typeof s)throw new Oe(i);if(a&&!c&&"wrapper"==ci(s))var c=new qn([],!0)}for(r=c?r:n;++r<n;){var u=ci(s=t[r]),l="wrapper"==u?si(s):o;c=l&&Ei(l[0])&&424==l[1]&&!l[4].length&&1==l[9]?c[ci(l[0])].apply(c,l[3]):1==s.length&&Ei(s)?c[u]():c.thru(s)}return function(){var e=arguments,r=e[0];if(c&&1==e.length&&Ga(r))return c.plant(r).value();for(var o=0,i=n?t[o].apply(this,e):r;++o<n;)i=t[o].call(this,i);return i}}))}function $o(e,t,n,i,a,s,c,u,l,f){var h=t&d,p=1&t,g=2&t,m=24&t,y=512&t,v=g?o:Lo(e);return function d(){for(var w=arguments.length,A=r(w),b=w;b--;)A[b]=arguments[b];if(m)var S=ui(d),E=function(e,t){for(var n=e.length,r=0;n--;)e[n]===t&&++r;return r}(A,S);if(i&&(A=Co(A,i,a,m)),s&&(A=Oo(A,s,c,m)),w-=E,m&&w<f){var T=dn(A,S);return Jo(e,t,$o,d.placeholder,n,A,T,u,l,f-w)}var x=p?n:this,k=g?x[e]:e;return w=A.length,u?A=function(e,t){var n=e.length,r=An(t.length,n),i=Bo(e);for(;r--;){var a=t[r];e[r]=Ai(a,n)?i[a]:o}return e}(A,u):y&&w>1&&A.reverse(),h&&l<w&&(A.length=l),this&&this!==gt&&this instanceof d&&(k=v||Lo(k)),k.apply(x,A)}}function Wo(e,t){return function(n,r){return function(e,t,n,r){return br(e,(function(e,o,i){t(r,n(e),o,i)})),r}(n,e,t(r),{})}}function qo(e,t){return function(n,r){var i;if(n===o&&r===o)return t;if(n!==o&&(i=n),r!==o){if(i===o)return r;"string"==typeof n||"string"==typeof r?(n=lo(n),r=lo(r)):(n=uo(n),r=uo(r)),i=e(n,r)}return i}}function Go(e){return oi((function(t){return t=Mt(t,Xt(li())),Yr((function(n){var r=this;return e(t,(function(e){return _t(e,r,n)}))}))}))}function Vo(e,t){var n=(t=t===o?" ":lo(t)).length;if(n<2)return n?Zr(t,e):t;var r=Zr(t,pt(e/pn(t)));retur
```

Comment:
```suggestion
      status: 'stopped',
```
When loading the app these aren't running right away